collector/sqs/pom.xml

```
@@ -42,7 +42,7 @@
     <dependency>
       <groupId>javax.xml.bind</groupId>
       <artifactId>jaxb-api</artifactId>
-      <version>2.2.12</version>
+      <version>2.4.0-b180725.0427</version>
```

I think this version is a stretch, may be:
```suggestion
      <version>2.3.1</version>
```